### DISCUSSION TITLE
"Command not found" all of a sudden

### DISCUSSION THREAD

**Discussion by RobCranfill** [wmerkens/vscode-circuitpython#16] [Q&A] [answered]
I've been using the new version (0.2.8) for a couple weeks now, and all of a sudden (today) none of it works. All I get is a "Command not found" error. 

![image](https://github.com/user-attachments/assets/49c6aafc-f58f-446c-9a09-bfb2e70fafd2)

Any thoughts? Any log files I could/should look at?

This is on Ubuntu 24.10. I've tried rebooting. It worked fine until today. I love this plugin, I gotta have it! ;-)

**Comment by wmerkens** [comment, marked as answer]
There is an update to VSC and NPM.

I did a rebuild of the extension. To take into account the changes, plus fix a minor issue, #14 

But I am still working on the library Manager and the unpacking of the bundle problem, so that you will have issues in that area; use circup at the CLI to manage libraries.

**Comment by RobCranfill** [comment]
I had trouble for the last couple days updating my VSC for some reason - low disk space? - but all is happy now, and the extention is working again. 

Thanks so much for working on this!
